src/data/faq.json

```
@@ -341,7 +377,8 @@
                         "textblock": [
                             "The Conference of Health Ministers has now passed a regulation on COVID 19 booster vaccinations. An additional booster vaccination is designed to help raise neutralizing antibody titers again, which may decline over time after basic immunization. This becomes particularly important if a weak/muted vaccine response is assumed or recognized after basic immunization in certain groups of vulnerable people.",
                             "The Corona-Warn-App (CWA) will notify the user to consider a booster vaccination if certain conditions are met. These conditions are based on the recommendations of the Paul Ehrlich Institute and the Permanent Vaccination Commission (STIKO) of the Robert Koch Institute (RKI).",
-                            "How long someone is effectively protected by a COVID 19 vaccination (basic immunization) depends on several factors and can therefore not be answered in general terms according to current knowledge. For this reason, the corresponding CWA rules for vaccination schedules and booster vaccinations are continuously reviewed and adapted in coordination with the RKI according to developments and new findings."
+                            "How long someone is effectively protected by a COVID 19 vaccination (basic immunization) depends on several factors and can therefore not be answered in general terms according to current knowledge. For this reason, the corresponding CWA rules for vaccination schedules and booster vaccinations are continuously reviewed and adapted in coordination with the RKI according to developments and new findings.",

```

```suggestion
                            "How long someone is effectively protected by a COVID 19 vaccination (basic immunization) depends on several factors and can therefore not be answered in general terms according to current knowledge. For this reason, the corresponding CWA rules for vaccination schedules and booster vaccinations are continually reviewed and adapted in coordination with the RKI according to developments and new findings.",
```
See https://www.dictionary.com/e/continually-vs-continuously/